docs/content/integrations/dbt.mdx

```diff
@@ -10,69 +10,52 @@ description: Dagster can orchestrate dbt alongside other technologies.
   <a href="/integrations/dbt-cloud">dbt Cloud with Dagster guide</a>!
 </Note>
 
-Dagster orchestrates dbt alongside other technologies, so you can combine dbt with Spark, Python, etc. in a single workflow. Dagster's [software-defined asset](/concepts/assets/software-defined-assets) abstractions make it simple to define data assets that depend on specific dbt models, or to define the computation required to compute the sources that your dbt models depend on. You could, for example:
+Dagster orchestrates dbt alongside other technologies, so you can schedule dbt with Spark, Python, etc. in a single data pipeline.
 
-- Run your dbt models after ingesting data into your data warehouse
-- Selectively materialize dbt models and their dependencies
+Dagster's [software-defined asset](/concepts/assets/software-defined-assets) approach allows it to understand dbt at the level of individual dbt models. This means that you can:
 
-Dagster has built-in support for loading dbt models, seeds, and snapshots as software-defined assets, enabling you to:
+- Use Dagster's UI or APIs to run subsets of your dbt models, seeds and snapshots.
+- Track failures, logs, and run history for individual dbt models, seeds, and snapshots.
+- Define dependencies between individual dbt models and other data assets. For example, put dbt models after the Fivetran-ingested table that they read from, or put a machine learning after the dbt models that its trained from.
 
-- Visualize and orchestrate a graph of dbt assets, and execute them with a single dbt invocation
-- Version your dbt models by their defining SQL code, allowing Dagster to indicate when a model has changed
-- View detailed historical metadata and logs for each asset
-- Define Python computations that depend directly on tables updated using dbt
-- Track data lineage through dbt and your other tools
+<!-- ![Dagster graph with dbt, Fivetran, and TensorFlow](/images/integrations/dbt/dagster-dbt-fivetran-tensorflow.png) -->
 
----
+<Image
+alt="Dagster graph with dbt, Fivetran, and TensorFlow"
+src="/images/integrations/dbt/dagster-dbt-fivetran-tensorflow.png"
+width={1834}
+height={1220}
+/>
 
-## Using dbt with Dagster
+```python file=/integrations/dbt/potemkin_dag_for_cover_image.py startafter=start endbefore=end
+fivetran_assets = dagster_fivetran.build_fivetran_assets(
+    connector_id="postgres",
+    table_names=["users", "orders"],
+)
 
-<DbtModelAssetExplanation />
+dbt_assets = dagster_dbt.load_assets_from_dbt_manifest("manifest.json")
 
-To learn how to load dbt models into Dagster as assets, check out the tutorial below or the quick version in the [dagster-dbt reference](/integrations/dbt/reference#loading-dbt-models-from-a-dbt-project).
+
+@asset(compute_kind="tensorflow", non_argument_deps={"daily_order_summary"})
+def predicted_orders():
+    ...
+```
```

Comment:
Having the image and code block up against each other in the rendered page looks odd. Can we add some copy between these things to make it look less cluttered?

Ex:

```
Define dependencies between individual dbt models and other data assets. For example, put dbt models after the Fivetran-ingested table that they read from, or put a machine learning after the dbt models that it's trained from:

[code example]

Which could produce an asset graph like the following:

[image]
```

<img width="660" alt="Screen Shot 2023-06-26 at 11 15 21 AM" src="https://github.com/dagster-io/dagster/assets/16615212/50fff5f4-ec0d-4db0-a86f-1e7e2ef73699">

Reply:
Yeah it does look odd. I was hoping to get the image to show up above the fold. I'm pushing up a version that includes text in between but puts the image first.  Let me know what you think.

Reply:
It looks great, thanks so much for making this change :)